-------------------------------------------------------------------------------
Test set: restassured.ValidationTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.483 s <<< FAILURE! - in restassured.ValidationTest
updateObjects(restassured.ValidationTest)  Time elapsed: 2.897 s  <<< FAILURE!
java.lang.RuntimeException: 
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "error" (class com.example.model.ResponseItem), not marked as ignorable (5 known properties: "data", "name", "updatedAt", "id", "createdAt"])
 at [Source: (String)"{"error":"7 is a reserved id and the data object of it cannot be overridden. You can create a new object via POST request and use new generated by id from it to send a PUT request."}"; line: 1, column: 11] (through reference chain: com.example.model.ResponseItem["error"])
	at restassured.ValidationTest.updateObjects(ValidationTest.java:185)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: 
Unrecognized field "error" (class com.example.model.ResponseItem), not marked as ignorable (5 known properties: "data", "name", "updatedAt", "id", "createdAt"])
 at [Source: (String)"{"error":"7 is a reserved id and the data object of it cannot be overridden. You can create a new object via POST request and use new generated by id from it to send a PUT request."}"; line: 1, column: 11] (through reference chain: com.example.model.ResponseItem["error"])
	at restassured.ValidationTest.updateObjects(ValidationTest.java:185)

